src/mavedb/routers/taxonomies.py

```diff
@@ -0,0 +1,131 @@
+from typing import Any, List
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy import func, or_
+from sqlalchemy.orm import Session
+
+from mavedb import deps
+from mavedb.models.taxonomy import Taxonomy
+from mavedb.view_models import taxonomy
+from mavedb.view_models.search import TextSearch
+
+router = APIRouter(
+    prefix='/api/v1/taxonomies', tags=['taxonomies'], responses={404: {'description': 'Not found'}}
+)
+
+@router.get("/concatenateTaxonomyList", status_code=200, response_model=List[str], responses={404: {}})
+def concatenate_taxonomy_tax_id_organism_names_common_name(
+    *,
+    db: Session = Depends(deps.get_db),
+) -> Any:
+    """
+    List distinct species names, in alphabetical order.
+    """
+
+    items = db.query(Taxonomy).all()
+    concatenates = []
+    for i in items:
+        if i.common_name != 'NULL' and not None:
```

Comment:
what is the `and not None` part of this clause doing? should it be `i.common_name != 'NULL' and i.common_name is not None`?